tests/integration_tests/build/test_coverage.py

```diff
@@ -18,9 +18,9 @@
 # Checkout the cpuid crate. In the future other
 # differences may appear.
 if utils.is_io_uring_supported():
-    COVERAGE_DICT = {"Intel": 82.99, "AMD": 82.31, "ARM": 82.51}
+    COVERAGE_DICT = {"Intel": 77.00, "AMD": 75.44, "ARM": 72.05}
```

Comment:
I cover a bit of this issue under my PR description with:

> ### Code coverage
> 
> Code coverage is significantly reduced in large part due to the quantity of generated code for CPUID registers, this can be seen in the report ([coverage-report-cpuid-latest.zip](https://github.com/firecracker-microvm/firecracker/files/10239373/coverage-report-cpuid-latest.zip)) as there ~1000 lines of uncovered generated code under src/cpuid/intel/registers.rs.
> 
> Generated doc tests being excluded from code coverage makes higher coverage difficult here (including doc tests in coverage is a nightly feature).

As a result of not being able to include doctests in coverage and not being able to exclude code (since `#[no_coverage]` is unstable) I think it is unavoidable we will see a significant drop in coverage here. This can be somewhat mitigated with additional tests but this may only be able to pull the coverage back up a percent or two.

Reply:
any idea when including doc tests being included in code coverage will hit stable?

Reply:
I created an issue on this https://github.com/firecracker-microvm/firecracker/issues/3206, this is blocked by https://github.com/rust-lang/rust/issues/56925

Reply:
Could we actually use the unstable flag just for gathering code coverage?